modes/info/evil-collection-info.el

```diff
@@ -53,7 +53,7 @@
     " " 'Info-scroll-up
     (kbd "C-]") 'Info-follow-nearest-node
     (kbd "DEL") 'Info-scroll-down
-    (kbd "C-i") 'Info-history-forward
+    (kbd "g TAB") 'Info-history-forward
```

Comment:
you bind "g TAB" here, but leave "C-o" for Info-history-back, I think it's also a kind of inconsistency, because I think people tend to think "C-i" is the reverse version of "C-o". what do you think about bind "C-i/o" to Info-history-forward/back, and use "g C-i/o" for the evil-jump-foreward/backward?

Reply:
Does it make sense to keep C-i for GUIs?

Reply:
IMHO it's more sensible to either keep C-i and C-o together, or remap them together and set a fallback like "g C-i/o" to the original evil-jump-foreward/backward, no matter for  GUI or TTY

Reply:
`C-i` is TAB. I want `TAB` to behave similarly to `<tab>` for terminal compatibility.

Reply:
there is a variable evil-want-C-i-jump, and its default is t, which  means TAB shouldn't behave similarly to <tab> by default, add a condition seems better

Reply:
evil-collection ignores evil-want-C-i-jump because it has its own bindings.

Reply:
Thanks for your illustration! The consistency in my mind is about <kbd>C-i</kbd> with <kbd>C-o</kbd>, but not with <kbd>\<tab\></kbd>, and unfortunately terminal can't make use of <kbd>\<tab\></kbd>. It looks good to me, thanks for your work!